### PR TITLE
Added a few names to en.json

### PR DESCRIPTION
**Pull request by Thomasasia**
#### Summary
```SUMMARY: Content "Added some extra possible names"```
#### Purpose of change
I thought of some names that I thought would go well with this game, so I added them. Nothing Major.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
These seem to be normal names that can be added just fine.

File: data/names/en.json [hunk 1]
```diff
@@ -2515,6 +2517,7 @@
 {"usage": "city", "name": "New Sharon"},
 {"usage": "city", "name": "New Shoreham"},
 {"usage": "city", "name": "New Sweden"},
+{"usage": "city", "name": "New Vegas"},
```
Comment (anothersimulacrum) [inline data/names/en.json]: These are generally based on real city names - is there a New Vegas somewhere?

Comment (Thomasasia) [reply data/names/en.json]: > These are generally based on real city names - is there a New Vegas somewhere?

There most likely is not. It's a reference to [A classic post apocalyptic video game of the same name](https://en.wikipedia.org/wiki/Fallout:_New_Vegas).

Comment (I-am-Erk) [reply data/names/en.json]: Let's keep the place names to real ones, it's best we don't go down the road of allowing references in this particular context.

Comment (Thomasasia) [reply data/names/en.json]: > Let's keep the place names to real ones, it's best we don't go down the road of allowing references in this particular context.

I gotcha, I'll get rid of that one.

All of the other places are real, so they should be good then.

**Pull request merged**